### PR TITLE
[OBSDEF-9795] update readme

### PR DESCRIPTION
**Pull request by aoberezin**
Update readme for charts-bin

### CODE REVIEW

**Review by nikolay-t — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,108 @@
...
-For sources please refer to [EMCECS/Charts repo](https://github.com/EMCECS/charts)
+To add this repository to your local Helm installation:
+
+> *_NOTE: Currently, this is a private repository, so use the private repo instructions below. Public repo options will be available once we go public._*
+
+This repo is for binary files only
+
+For sources please refer to [EMCECS/Charts repo](https://github.com/EMCECS/charts) 
+
+## Adding the ECS Repository
```
Comment (eddiepavkovic) [inline README.md:13]:
```suggestion
## Adding the ObjectScale Repository
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,108 @@
...
+
+```bash
+$ helm repo add ecs https://emcecs.github.io/charts-bin
```
Comment (eddiepavkovic) [inline README.md:16]:
```suggestion
$ helm repo add objectscale https://emcecs.github.io/charts-bin
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,108 @@
...
+$ helm repo update
+```
+
+## Adding private Helm repository and Docker registries
+
+In this development phase, both the Helm repository and referenced Docker registries are private. Before installing charts from this repository, you must add private credentials to Helm and Kubernetes for Github and Docker Hub, respectively.
+
```
Comment (eddiepavkovic) [inline README.md:23]: its public now right?  so we can remove this.

Comment (eddiepavkovic) [reply README.md:23]: they only need the token to get to a specific version.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,108 @@
...
+  - Click "Create new token" at the top right. This will select all permissions for repositories.
+  - Add a token description
+  - Select the "repo" permissions checkbox
+  - Copy the token for use in the next step
+
+2. Create the repository using your token:
+
+Master branch (Option A):
+```bash
+$ helm repo add ecs \
```
Comment (eddiepavkovic) [inline README.md:38]:
```suggestion
$ helm repo add objectscale \
```

File: README.md [hunk 5]
```diff
@@ -1,6 +1,108 @@
...
+  "https://MY_PRIVATE_TOKEN@raw.githubusercontent.com/emcecs/charts-bin/master/docs"
+
+or 
+
+$ helm repo add ecs https://emcecs.github.io/charts-bin
```
Comment (eddiepavkovic) [inline README.md:43]:
```suggestion
$ helm repo add objectscale https://emcecs.github.io/charts-bin
```

File: README.md [hunk 6]
```diff
@@ -1,6 +1,108 @@
...
+
+$ helm repo update
+```
+
+Specific version (Option B):
+```bash
+$ helm repo add ecs \
```
Comment (eddiepavkovic) [inline README.md:50]:
```suggestion
$ helm repo add objectscale \
```

File: README.md [hunk 7]
```diff
@@ -1,6 +1,108 @@
...
+  - Add a token description
+  - Select the "repo" permissions checkbox
+  - Copy the token for use in the next step
+
+2. Create the repository using your token:
+
+Master branch (Option A):
+```bash
+$ helm repo add ecs \
+  "https://MY_PRIVATE_TOKEN@raw.githubusercontent.com/emcecs/charts-bin/master/docs"
```
Comment (eddiepavkovic) [inline README.md:39]:
```suggestion
  "https://raw.githubusercontent.com/emcecs/charts-bin/master/docs"
```
private token not needed for public repo

File: README.md [hunk 8]
```diff
@@ -1,6 +1,108 @@
...
+
+### Add Private Github-Hosted Helm Repository
+
+1. Create an authentication token in Github with read access to the charts repository.
+
+  - Navigate to your account [personal access tokens](https://github.com/settings/tokens)
+  - Click "Create new token" at the top right. This will select all permissions for repositories.
+  - Add a token description
+  - Select the "repo" permissions checkbox
+  - Copy the token for use in the next step
```
Comment (eddiepavkovic) [inline README.md:32]: not needed for public repo.  unless it goes private again.

File: README.md [hunk 9]
```diff
@@ -1,6 +1,108 @@
...
+```bash
+$ helm repo add ecs \
+  "https://raw.githubusercontent.com/emcecs/charts-bin/<version tag>/docs"
+$ helm repo update
+```
+
+3. Ensure that you can see the ECS charts
+
+```bash
+$ helm search ecs
```
Comment (eddiepavkovic) [inline README.md:58]:
```suggestion
$ helm search objectscale
```

File: README.md [hunk 10]
```diff
@@ -1,6 +1,108 @@
...
+ecs/objectscale-manager       	0.77.0       	0.77.0     	Dell EMC ObjectScale is highly scalable, and hi...
+ecs/objectscale-portal        	0.77.0       	0.77.0     	ObjectScale Portal                                
+ecs/objectscale-vsphere       	0.77.0       	0.77.0     	ObjectScale VMware vSphere Plugin                 
+ecs/pravega-operator          	0.5.2        	0.5.2      	Pravega Operator Helm chart for Kubernetes        
+ecs/service-pod               	2.77.0       	2.77.0     	A Helm chart for Dell EMC Service Pod             
+ecs/sonobuoy                  	0.16.6       	0.16.6     	A Helm chart for sonobuoy                         
+ecs/srs-gateway               	1.2.0        	1.2.0      	A Helm chart for Dell EMC SRS Gateway Custom Re...
+ecs/statefuldaemonset-operator	0.77.0       	0.77.0     	StatefulDaemonSet operator deploys operator pod...
+ecs/supportassist             	2.76.0       	2.76.0     	Helm chart for Dell SupportAssist ESE             
+ecs/zookeeper-operator        	0.28.0       	0.28.0     	Zookeeper operator deploys a custom resource fo...
```
Comment (eddiepavkovic) [inline README.md:90]: update to show objectscale

Comment (aoberezin) [reply README.md:90]: the same renaming should be in the original charts readme then :)

File: README.md (hunk 3 above)
Comment (aoberezin) [reply README.md:23]: Actually I'm able to get specific version without token

File: README.md (hunk 7 above)
Comment (aoberezin) [reply README.md]: no need in this section then, if we already have 
`helm repo add ecs https://emcecs.github.io/charts-bin`

File: README.md (hunk 1 above)
Comment (aoberezin) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (aoberezin) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (aoberezin) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (aoberezin) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (aoberezin) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (aoberezin) [reply README.md]: removed this section

File: README.md (hunk 9 above)
Comment (aoberezin) [reply README.md]: done

File: README.md (hunk 10 above)
Comment (aoberezin) [reply README.md]: done

**Review by antonbassin — APPROVED** [review]

**Pull request merged**